app/views/download.scala.html

```diff
@@ -107,7 +108,22 @@ <h3 id="older-versions">Previous releases</h3>
       <table class="releases">
       @releases.previous.groupBy(_.version.slice(0, 3)).toSeq.sortBy(_._1).reverse.map { group =>
         <tbody class="version">
-          <tr><th colspan="3"><h4>@group._1</h4></th></tr>
+          <tr>
+            <th colspan="3">
+              <h4 class="previousVersionHeading">@group._1</h4>
+              <a href="@{
+                    if(group._1.toDouble >= 1.2) {
```

Comment:
Converting to double is not a nice way to compare versions. I would write something like:

```scala
// check if version v1 >= v2
def versionAtLeast(v1: String, v2: String): Boolean = {
  import math.Ordering.Implicits._
  v1.split('.').toSeq > v2.split('.').toSeq
}
```
The imported ordering lets us compare versions as lists of string numbers.